examples/workflows/plot_simulate_somato.py

```diff
@@ -159,7 +159,7 @@
 # proximal drive. In order to understand the physiological implications of
 # proximal and distal drive as well as the general process used to articulate
 # a sequence of exogenous drive for simulating evoked responses, see the
-# `HNN ERP tutorial`_. Note that setting ``sync_within_trial=True`` creates
+# `HNN ERP tutorial`_. Note that setting ``n_drive_cells=1`` creates
```

Comment:
It really creates just one drive that's "shared" by all granular cells, right? So it's 'drive', not 'drives', and the 'synchronous input' discussion is no longer relevant/needed.

Reply:
Correct, however, I think this discussion is still useful as it describes the mechanism through which a user can synchronize drive across cells.